### PR TITLE
[KB-246] Z-score Normalization (standard scaler)

### PR DESCRIPTION
**Pull request by Ivikhostrup**
Not a long section. Perhaps the only thing I am really concerned about is the cite. I've put it at the end because the section is so short, but mayber you'd prefer following the convention of saying "Its based on.."?

### CODE REVIEW

File: report_thesis/src/sections/background.tex [hunk 1]
```diff
@@ -1,7 +1,16 @@
 \section{Background}\label{sec:background}
 \subsection{Preprocessing}
 
-\subsubsection{Standard Scaler}
+\subsubsection{Z-score Normalization}
+Z-score normalization, also known as standard score normalization, transforms the data into a distribution with a mean of zero and a standard deviation of one.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:5]:
```suggestion
Z-score normalization, also known as standard score normalization, transforms data to have a mean of zero and a standard deviation of one.
```

File: report_thesis/src/sections/background.tex [hunk 2]
```diff
@@ -1,7 +1,16 @@
 \section{Background}\label{sec:background}
 \subsection{Preprocessing}
 
-\subsubsection{Standard Scaler}
+\subsubsection{Z-score Normalization}
+Z-score normalization, also known as standard score normalization, transforms the data into a distribution with a mean of zero and a standard deviation of one. 
+This normalization is useful when the actual minimum and maximum of a feature are unknown or when there are outliers that may skew the distribution significantly.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:6]:
```suggestion
This technique is useful when the actual minimum and maximum of a feature are unknown or when outliers may significantly skew the distribution.
```

File: report_thesis/src/sections/background.tex [hunk 3]
```diff
@@ -1,7 +1,16 @@
 \section{Background}\label{sec:background}
 \subsection{Preprocessing}
 
-\subsubsection{Standard Scaler}
+\subsubsection{Z-score Normalization}
+Z-score normalization, also known as standard score normalization, transforms the data into a distribution with a mean of zero and a standard deviation of one. 
+This normalization is useful when the actual minimum and maximum of a feature are unknown or when there are outliers that may skew the distribution significantly. 
+The formula for Z-score normalization is given by:
+
+$$ v' = \frac{v - \overline{F}}{\sigma_F} $$
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:9]:
```suggestion
$$ 
v' = \frac{v - \overline{F}}{\sigma_F},
$$
```

File: report_thesis/src/sections/background.tex [hunk 4]
```diff
@@ -1,7 +1,16 @@
...
+
+where $v$ is the original value, $\overline{F}$ is the mean of the feature $F$, and $\sigma_F$ is the standard deviation of $F$. 
+By transforming the data using the Z-score, each value reflects the number of standard deviations away from the mean.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:12]:
```suggestion
By transforming the data using the Z-score, each value reflects its distance from the mean in terms of standard deviations.
```

File: report_thesis/src/sections/background.tex [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
+\subsubsection{Z-score Normalization}
+Z-score normalization, also known as standard score normalization, transforms the data into a distribution with a mean of zero and a standard deviation of one. 
+This normalization is useful when the actual minimum and maximum of a feature are unknown or when there are outliers that may skew the distribution significantly. 
+The formula for Z-score normalization is given by:
+
+$$ v' = \frac{v - \overline{F}}{\sigma_F} $$
+
+where $v$ is the original value, $\overline{F}$ is the mean of the feature $F$, and $\sigma_F$ is the standard deviation of $F$. 
+By transforming the data using the Z-score, each value reflects the number of standard deviations away from the mean. 
+This method is helpful in environments where the data needs to be standardized to fit a normal distribution, making it easier to compare differences in data points~\cite{dataminingConcepts}.
```
Comment (Pattrigue) [inline report_thesis/src/sections/background.tex:13]:
```suggestion
Z-score normalization is particularly advantageous in scenarios where data features have different units or scales, or when preparing data for algorithms that assume normally distributed inputs~\cite{dataminingConcepts}.
```

**Review by Pattrigue — CHANGES_REQUESTED** [review]

**Review by Pattrigue — APPROVED** [review]

File: report_thesis/src/sections/background.tex [hunk 6]
```diff
@@ -1,7 +1,18 @@
 \section{Background}\label{sec:background}
 \subsection{Preprocessing}
 
-\subsubsection{Standard Scaler}
+\subsubsection{Z-score Normalization}
+Z-score normalization, also known as standard score normalization, transforms data to have a mean of zero and a standard deviation of one.
```
Comment (chhoumann) [inline report_thesis/src/sections/background.tex:5]: Known as standardization, no? Never heard of `standard score normalization`.

**Review by chhoumann — APPROVED** [review]

**Pull request merged**